dkg/dkgpb/v1/lockhashk1.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package dkg.dkgpb.v1;
+
+option go_package = "github.com/obolnetwork/charon/dkg/dkgpb/v1";
+
+message MsgLockHashK1Sig {
+  bytes hash_signature = 1;
```

Comment:
nit: rename to `signature` since "hash" stutters a but since it is already in type name.